src/resolve/resolved-conf.c

```diff
@@ -599,3 +599,37 @@ int manager_parse_config_file(Manager *m) {
         return 0;
 
 }
+
+int config_parse_dns_cache_size(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        Manager *m = ASSERT_PTR(userdata);
+        int ret, val;
+
+        assert(filename);
+        assert(rvalue);
+
+        if (isempty(rvalue)) {
+                m->cache_size = CACHE_MAX_DEFAULT;
+                return 0;
+        }
+
+        ret = safe_atoi(rvalue, &val);
```

Comment:
as per coding style we use "ret" for naming return parameters of functions (see https://systemd.io/CODING_STYLE/#code-organization-and-semantics), hence please don't use it for a local variable here. Moreover we always call the variable we place error codes in `int r`. Hnece please stick to that here too, i.e. rename `ret` → `r`

Reply:
Apologies, I didn't notice that particular standard, updated the block to following the style guide